### DISCUSSION TITLE
Support better multi-root workspace configuration

### DISCUSSION THREAD

**Discussion by agreedSkiing** [redhat-developer/vscode-yaml#883] [Ideas]
I work in a project where we support a platform that has configuration that uses the default redhat.vscode-yaml for formatting with their settings defined in each .vscode/settings.json and some newer configuration projects that use prettier for formatting and here is where I run into a problem.

When opening multiple projects in a VSCode which creates a multi-root workspace the redhat.vscode-yaml configuration stops working ☹, and since there is no standard set in our projects the have different indentations and use of “ and ‘. 
![bild](https://user-images.githubusercontent.com/93997033/216323966-e9bc1b3f-3af6-4bfe-8c16-cafb52beda23.png)
So it would be a great improvement to support multi-root workspaces in a wider scope, since I can’t have a one config fits all here.

**Comment by gorkem** [comment]
Sounds like a reasonable feature to ask :) created #927
